bindings.go

```diff
@@ -16,21 +19,29 @@ var readFuncs = &readFuncsMap{funcs: make(map[int]ReadFunc)}
 
 // Parse is a shortcut for parsing bytes of source code,
 // returns root node
-func Parse(content []byte, lang *Language) *Node {
+func Parse(ctx context.Context, content []byte, lang *Language) (*Node, error) {
```

Comment:
**Question:** Is it worth adding a new `Parse` method that takes a context so as not to break all dependents? Otherwise I agree that requiring context is the best design of this method.

Reply:
I was prepared to version bump this package in order to deal with this, but I read the Go module versioning documentation and it says that a v0 package (which this library is) can break its API whenever it wants, so I didn't do it.